File: Napakalaki/lib/ejemplo_main.rb
# encoding: utf-8

require_relative 'napakalaki'
require_relative 'game_tester'

module NapakalakiGame

  # class EjemploMain: partida de ejemplo para el juego
  class EjemploMain
    def prueba
      test = Test::GameTester.instance
      game = Napakalaki.instance
      
      #Se prueba el juego con 2 jugadores
      test.play(game, 2);
    end
      
  end # EjemploMain
  
  e = EjemploMain.new
  e.prueba

end # NapakalakiGame

File: Napakalaki/src/NapakalakiGame/EjemploMain.java
/**
 * EjemploMain.java
 */

package NapakalakiGame;

import Test.GameTester;

/**
 * Prueba del desarrollo de una partida.
 */
public class EjemploMain {

    public static void main(String[] args) {
      Napakalaki game = Napakalaki.getInstance();
      GameTester test = GameTester.getInstance();
      
      // Poner el numero de jugadores con el que se quiera probar
      test.play(game, 2); 
              
    }
}
